app/javascript/src/components/Dashboard/Notes/index.jsx

```diff
@@ -66,11 +67,13 @@ const Notes = () => {
                 disabled={!selectedNoteIds.length}
                 icon={Delete}
                 label="Delete"
+                size="small"
                 onClick={() => setShowDeleteAlert(true)}
               />
             }
           />
           <Table
+            rowSelection
```

Comment:
@yedhink checkbox not working in the table even after adding `rowSelection`. Also  `rowSelection` is not mentioned in docs its just mentioned in a example below in table section. 
Ref: https://neetoui.onrender.com/iframe.html?viewMode=docs&id=components-table--table-without-checkbox#table-without-checkbox

Reply:
As discussed in the call, check the neetoUI table and raise an issue there if need be.